autoload/lsp/completion.vim

```diff
@@ -90,7 +90,7 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
 
   # Get the keyword prefix before the current cursor column.
   var chcol = charcol('.')
-  var starttext = getline('.')[ : chcol - 2]
+  var starttext = chcol == 1 ? '' : getline('.')[ : chcol - 2]
```

Comment:
I tried various completion methods and found that the column number is always greater than 1.
Do you have an example that shows the condition under which the column is 1?